pkg/riff/commands/function.go

```diff
@@ -65,16 +66,28 @@ func FunctionCreate(buildpackBuilder core.Builder, fcTool *core.Client) *cobra.C
 				return err
 			}
 
-			if createFunctionOptions.Image == "" {
+			if createFunctionOptions.Image == "" || strings.HasPrefix(createFunctionOptions.Image, "_") {
```

Comment:
small nit, now that there is no `_:`
```suggestion
			if createFunctionOptions.Image == "" || strings.HasPrefix(createFunctionOptions.Image, "_/") {
```

Reply:
I'd rather keep it as is